remove-user-on-logout/pom.xml

```diff
@@ -1,5 +1,5 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
I reformatted this file - make sure to fix your IntelliJ config:
![image](https://github.com/user-attachments/assets/8c15edc2-76b3-44e6-8bb9-31c1b61c585a)

Reply:
@ingokuba Can you add that to our eclipse profile?
Reformatted it but that won't change that line. Continuation indent is 8...